assignment/README.md

```diff
@@ -53,13 +64,13 @@ most difficult about this project? How will you attempt to cope with
 these difficulties? For example, asynchronous behavior (ajax, events)
 are hard to use and think about. Global variables are a strategy for
 coping with that difficulty by breaking data out of the asynchronous
-context.
+context.  
+__Data__:  
+Generating the data raster/vector data may be time-consuming.  
+How to load georeferenced geoTiff images onto the map, while preserving the accuracy and making it zoomable? I'll try different plugins to solve this problem.  
+__Interaction__:  
+How to better deal with asynchronous behavior? I can start with log to console to have a better understanding of the sequence.
 
 ## Missing pieces
 
-We've only managed to scratch the surface of the available technologies
-by which you could construct an application. What use-cases haven't we covered
-that you think would be useful? What technologies not covered seem exciting to
-you (you don't necessarily have to fully understand what they're for,
-this is a chance for you to get our help interpreting a technology's
-purpose/usage).
+I really hope we can talk more about the server end of web development, especially retrieving data from server-end database and deployment.
```

Comment:
We'll probably have to talk a bit bout this independent from the rest of the class. If it proves too tricky, try to have in mind a less sophisticated alternative